Rakefile

```diff
@@ -5,6 +5,12 @@ Rake::TestTask.new(:test) do |t|
   t.libs << "test"
   t.libs << "lib"
   t.test_files = FileList["test/**/*_test.rb"]
+
+  # TODO: We should remove this line. See `puts` line below.
+  t.warning = false
+
+  puts "Reminder: Remove `t.warning = false` in Rakefile once graphql-ruby fixes all instances of"\
+    "`warning: instance variable @<ivar> not initialized` and `mismatched indentations`"
```

Comment:
I bumped rubies and started getting many hundreds of these warnings, coming from graphql-ruby. This is left as a TODO. It's a bit unfortunate, but not doing so makes test output unnecessarily long.

Reply:
Can confirm I am having the same behaviour in the test helper gem I am working on. If you find a solution I'd love to know it. For now I'm doing the same thing as you here. 😃 👍